js/PageLevelProgressIndicatorView.js

```diff
@@ -60,28 +56,19 @@ class PageLevelProgressIndicatorView extends Backbone.View {
       const visited = children.filter(child => child.get('_isVisited'));
       return Math.round(visited.length / children.length * 100);
     }
-    return 0
+    return 0;
   }
 
   render() {
+    this.checkCompletion();
+    this.checkAriaHidden();
     const data = this.getRenderData();
-    const template = Handlebars.templates[this.constructor.template];
-    this.$el.html(template(data));
-  }
-
-  getRenderData() {
-    const data = this.model.toJSON();
-    data.ariaLabel = this.ariaLabel;
-    data.type = this.type;
-    return data;
+    const Component = templates.pageLevelProgressIndicator;
+    ReactDOM.render(<Component {...data} />, this.el);
   }
 
   refresh() {
```

Comment:
Do we need to have the refresh function if all it does is call render? Could we not call render directly?